Week7/VendingMachine.java

```diff
@@ -0,0 +1,64 @@
+
+public class VendingMachine
+{
+	double balance; // amount of money inserted for a purchase
```

Comment:
You could give more meaningful names to your variables to make reading the code easy, by naming it "MoneyInserted" or "CashReceived" instead of a comment. Comment helps in explaining this variable at the point of creation. but later on when one is reading your code, one might forget what that variable stands for.